### DISCUSSION TITLE
Sending Proxy username and password on JDBC

### DISCUSSION THREAD

**Discussion by emoran** [prestodb/presto#15081] [General]
Hi all, 

I'm trying to use the JDBC driver to get connected to presto, but when I run the process I get:

` Error Cause java.io.UncheckedIOException: java.io.IOException: Failed to authenticate with proxy`

I see in the documentation we can pass the property **httpProxy** but no how to pass authentication to it.

Hope someone can give me some insights. Thanks.

**Comment by fornaix** [comment]
```java
DriverManager.getConnection(url, userName, password);
```
use this api to pass basic access authentication
